### DISCUSSION TITLE
clickhouse-keeper question

### DISCUSSION THREAD

**Discussion by hert525** [ClickHouse/ClickHouse#65139] [Q&A]
"My current ClickHouse version is 22.6.4.35. When writing to the ReplicatedReplacingMergeTree table created with clickhouse-keeper, occasional errors like 'Table is in readonly mode' and 'connection loss' occur. How can I troubleshoot these issues? Are there any recommended clickhouse-keeper parameters that can help avoid such problems?

**Comment by Algunenano** [comment]
Check the logs of CH and keeper before you get the message to see why it happened

**Comment by leeningli** [comment]
is insert too frequently, causing the zk load to be too high
